### ISSUE TITLE
Create dataset:tensor-conv-mobilenets-v1-0.75-160

### ISSUE THREAD

**Issue by psyhtest** [ctuning/ck-nntest#19]
For benchmarking, Arm use two models of the MobileNets-v1 family: `1.0-224` (baseline) and `0.75-160` (reduced). We should create `dataset:tensor-conv-mobilenets-v1-0.75-160` similar to [`dataset:tensor-conv-mobilenets-v1-1.0-224`](https://github.com/ctuning/ck-nntest/commit/4db265b7b457ef2c3eda05d03c69cc146d716aef) with **all** the shapes (including the 3x3 convolution layer with stride 2 and logits layer).

**Comment by Chunosov** [comment]
PR https://github.com/ctuning/ck-nntest/pull/20

**Comment by psyhtest** [comment]
Thanks!
